test_infer_request.py renamed to test_sync_infer_request.py and new f…

Bhaskar365 opened the pull request:
Hello, this is my first pull request in open source contributions , so please forgive me if I have made any silly errors. As per your request, I just created test_infer_request.py which was renamed to test_sync_infer_request.py and new file 
created called test_async_infer_request.py in directory: **openvino/src/bindings/python/tests/test_runtime/** containing test of AsyncInferQueue. If you think something is missing and needs addition or anything else is request , please leave a comment below the PR and thank you for your time.

Thanks Bhaskar

mlukasze commented:
build_jenkins

p-wysocki commented on src/bindings/python/tests/test_runtime/test_sync_infer_request.py:
```diff
@@ -15,7 +15,7 @@
     CompiledModel,
     InferRequest,
-    AsyncInferQueue,
+    # AsyncInferQueue,
```
Could you please remove it completely? We tend to avoid having commented out code.

akuporos replied:
And remove/uncomment all commented tests

Bhaskar365 commented:
I have made the changes and pushed the commits , please check whether pull request has been successfully added or not? Waiting for your response.

Thanks Bhaskar

akuporos commented:
@Bhaskar365 could you please merge master to your branch and resolve conflict?

akuporos commented on src/bindings/python/tests/test_runtime/test_sync_infer_request.py:
```diff
@@ -36,7 +35,8 @@ def create_model_with_memory(input_shape, data_type):
     res = ops.result(add, "res")
-    model = Model(results=[res], sinks=[node], parameters=[input_data], name="name")
+    model = Model(results=[res], sinks=[node],
+                  parameters=[input_data], name="name")
```
Could you please also remove style changes like this in this file?

Bhaskar365 replied:
Hi @akuporos , 
     Hope you are well. I just made the changes you requested . Please check whether everything is ok as per your request . Please comment if any changes are required.

Thanks Bhaskar

akuporos commented:
Hi @Bhaskar365,

Looks like something went wrong when you merged master. And there is still present conflicting file.

Bhaskar365 commented:
Hello @akuporos and Team, 
        I just execute git merge upstream/master . Please check and tell me whether it is done correctly this time.

Regards Bhaskar

mlukasze commented:
hey @Bhaskar365 
please resolve conflicts before we go further

Bhaskar365 commented:
> hey @Bhaskar365 please resolve conflicts before we go further

Hello mr @mlukasze ,  I found this conflict initially in my terminal - 

_C:\Users\Stark\Pictures\PythonOpenSourceContribution\openvino> git merge upstream/master
Auto-merging src/bindings/python/tests/test_runtime/test_sync_infer_request.py
CONFLICT (content): Merge conflict in src/bindings/python/tests/test_runtime/test_sync_infer_request.py
Automatic merge failed; fix conflicts and then commit the result.
PS C:\Users\Stark\Pictures\PythonOpenSourceContribution\openvino> git merge upstream/master
error: Merging is not possible because you have unmerged files.
hint: Fix them up in the work tree, and then use 'git add/rm <file>'
hint: as appropriate to mark resolution and make a commit.
fatal: Exiting because of an unresolved conflict.
PS C:\Users\Stark\Pictures\PythonOpenSourceContribution\openvino> I removed some symbols from main file which was giving merge issues and then the second time I put merge command this came_

and then I removed all this 
_<<<<<<< HEAD
# your changes
=======
# changes from upstream/master
>>>>>>> upstream/master_

and then I put this command - _git add src/bindings/python/tests/test_runtime/test_sync_infer_request.py_
after which no more conflicts were reported .

 So, please tell me what other conflicts are there to be resolved . I apologize that I am taking this long to resolve all this, I have never contributed to open source before, so I am learning as I am doing and checking the mistakes on the web for help.

Regards Bhaskar

p-wysocki commented:
Hey @Bhaskar365, don't worry about not getting it right the first time, we're happy to help and teach. :) 

Merge to master went wrong, you can see it by how many files are changed - majority of them are not your changes. In order to get back to when it was still fine, you can reset your branch to the last commit **before** merging master:

`git reset --hard c90ca9ee57b73a6c5acbd4dddc0da53bea0ede24`

This will rewrite your branch back in time to that commit. After that you need to merge current OpenVINO master. In order to do that, you first run `git fetch`, which will download all branch content locally into `.git` directory in your project, but will make no changes to your branch:

`git fetch --all upstream/master`

Please note that `upstream` may be different for you - you need to check what name you gave to OpenVINO repository by using `git remote -v`.

Once fetching is done, you need to actually merge current OpenVINO master with your branch - in order to do that, you need to run:

`git merge upstream/master`

At this point you need to resolve any conflicts. Once you do it, all is left is to push your changes to the remote branch - that's when we'll se them here, in the Pull Request. In order to do that, run:

`git push origin/splitFilesBranch`

Since you did a `git reset`, this command will likely not work straight away because `reset` rewrites commit history, something Git doesn't like. You can overcome this by adding the flag `--force`. Again, this command also assumes that your fork in `git remote -v` is called `origin`.

Please let us know if you have any other questions and don't worry about git, we've all had to start somewhere and I myself remember having to run seemly random commands just to see what happens.

Bhaskar365 commented:
> Hey @Bhaskar365, don't worry about not getting it right the first time, we're happy to help and teach. :)
> 
> Merge to master went wrong, you can see it by how many files are changed - majority of them are not your changes. In order to get back to when it was still fine, you can reset your branch to the last commit **before** merging master:
> 
> `git reset --hard c90ca9ee57b73a6c5acbd4dddc0da53bea0ede24`
> 
> This will rewrite your branch back in time to that commit. After that you need to merge current OpenVINO master. In order to do that, you first run `git fetch`, which will download all branch content locally into `.git` directory in your project, but will make no changes to your branch:
> 
> `git fetch --all upstream/master`
> 
> Please note that `upstream` may be different for you - you need to check what name you gave to OpenVINO repository by using `git remote -v`.
> 
> Once fetching is done, you need to actually merge current OpenVINO master with your branch - in order to do that, you need to run:
> 
> `git merge upstream/master`
> 
> At this point you need to resolve any conflicts. Once you do it, all is left is to push your changes to the remote branch - that's when we'll see them here, in the Pull Request. In order to do that, run:
> 
> `git push origin/splitFilesBranch`
> 
> Since you did a `git reset`, this command will likely not work straight away because `reset` rewrites commit history, something Git doesn't like. You can overcome this by adding the flag `--force`. Again, this command also assumes that your fork in `git remote -v` is called `origin`.
> 
> Please let us know if you have any other questions and don't worry about git, we've all had to start somewhere and I myself remember having to run seemly random commands just to see what happens.

Thank you Mr @p-wysocki for giving advice on the underlying problem and required steps and I am delighted that you and everyone else is giving me a chance to do something on a open source project. Well let me follow your code and tell you once merge is done. 

One more thing I have a put git remote -v I did get this in my terminal - 
_C:\Users\Stark\Pictures\PythonOpenSourceContribution\openvino> git remote -v
origin  https://github.com/Bhaskar365/openvino.git (fetch)
origin  https://github.com/Bhaskar365/openvino.git (push)
upstream        https://github.com/openvinotoolkit/openvino.git (fetch)
upstream        https://github.com/openvinotoolkit/openvino.git (push) 

 Please comment whether this remote list is correct or wrong so that my first step to resolve the conflict is in the right direction

akuporos commented:
Hello @Bhaskar365,

Yes, your upstream set is right.

Bhaskar365 commented:
Ok @akuporos , I will let you know once I have raised merge request

Bhaskar365 commented:
Hi Team, 
I have a problem which is this - 

_git fetch --all upstream/master
fatal: fetch --all does not take a repository argument
PS C:\Users\Stark\Pictures\PythonOpenSourceContribution\openvino> git fetch upstream/master      
fatal: 'upstream/master' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
PS C:\Users\Stark\Pictures\PythonOpenSourceContribution\openvino>_

I ran the git fetch command after git reset command , well this fetch command was working earlier properly . Is it the wrong directory I have selected or is it something else?

akuporos commented:
After `git reset --hard c90ca9ee57b73a6c5acbd4dddc0da53bea0ede24` I'd suggest `git push -f origin splitFilesBranch` -- to clean up your remote repo.
After that `git fetch` and all the stuff @p-wysocki described.

p-wysocki commented:
@Bhaskar365 

>PS C:\Users\Stark\Pictures\PythonOpenSourceContribution\openvino> git push upstream splitFilesBranch
remote: Permission to openvinotoolkit/openvino.git denied to Bhaskar365.
fatal: unable to access 'https://github.com/openvinotoolkit/openvino.git/': The requested URL returned error: 403

Please note that you're trying to push to `upstream`, which is main OpenVINO repository - it's protected from pushes, so it returns error code 403. You can only push to your fork, please run `git push origin splitFilesBranch`. Let us know it if works now!

> and one more thing if I enter git fetch --all upstream/master I get this error -
fatal: fetch --all does not take a repository argument

Please run `git pull upstream master`, it should work now.

Bhaskar365 commented:
Hello Mr @p-wysocki , I got everything up to date after pushing to origin branch . Please check whether what you wanted is done or something more is required..

Bhaskar365 commented:
Hi, well I tried git status and this came, maybe it might of some help - 

_git status
On branch splitFilesBranch
All conflicts fixed but you are still merging.
  (use "git commit" to conclude merge)        

Changes to be committed:
        modified:   .github/actions/setup_python/action.yml
        modified:   .github/workflows/linux.yml
        new file:   .github/workflows/linux_arm64.yml
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/convert_python_model_objects.rst      
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/cutting_model.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/embedding_preprocessing_computation.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/fp16_compression.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/model_optimizer_faq.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/setting_input_shapes.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/Convert_Model_From_ONNX.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/Convert_Model_From_Paddle.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/Convert_Model_From_PyTorch.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/Convert_Model_From_TensorFlow.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/Convert_Model_From_TensorFlow_Lite.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_AttentionOCR_From_Tensorflow.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_BERT_From_Tensorflow.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_Bert_ner.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_CRNN_From_Tensorflow.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_Cascade_RCNN_res101.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_DeepSpeech_From_Tensorflow.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_EfficientDet_Models.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_F3Net.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_FaceNet_From_Tensorflow.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_Faster_RCNN.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_GNMT_From_Tensorflow.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_GPT2.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_Mask_RCNN.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_NCF_From_Tensorflow.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_Object_Detection_API_Models.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_QuartzNet.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_RCAN.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_RNNT.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_RetinaNet_From_Tensorflow.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_Slim_Library_Models.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_WideAndDeep_Family_Models.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_XLNet_From_Tensorflow.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_YOLACT.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_YOLO_From_Tensorflow.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition/legacy_conversion_api/supported_model_formats/convert_model_tutorials/Convert_lm_1b_From_Tensorflow.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/pot_api_reference.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/pot_cli.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/pot_cli/configuration_file_description.rst    
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/pot_cli/simplified_mode.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/pot_examples.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/pot_examples/pot_api_examples.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/pot_examples/pot_api_examples/pot_example_3d_segmentation.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/pot_examples/pot_api_examples/pot_example_classification.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/pot_examples/pot_api_examples/pot_example_face_detection.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/pot_examples/pot_api_examples/pot_example_object_detection.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/pot_examples/pot_api_examples/pot_example_segmentation.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/pot_examples/pot_api_examples/pot_example_speech.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/pot_examples/pot_cli_example.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/pot_faq.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/protecting_model.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/quantization_best_practices.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/quantization_best_practices/saturation_issue.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/quantizing_models.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/quantizing_models/default_quantization_algorithm.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/quantizing_models_with_accuracy.rst
        modified:   docs/articles_en/documentation/openvino_legacy_features/post_training_optimization_tool/quantizing_models_with_accuracy/accuracy_aware_algorithm.rst
        modified:   docs/documentation_build_instructions.md
        modified:   docs/sphinx_setup/api/ie_python_api/api.rst
        modified:   src/bindings/python/src/openvino/frontend/tensorflow/utils.py
        modified:   src/bindings/python/src/pyopenvino/graph/model.cpp
        modified:   src/bindings/python/tests/test_runtime/test_model.py
        modified:   src/bindings/python/tests/test_runtime/test_sync_infer_request.py
        modified:   src/bindings/python/tests/utils/helpers.py
        modified:   src/common/snippets/include/snippets/generator.hpp
        modified:   src/common/snippets/include/snippets/lowered/linear_ir.hpp
        new file:   src/common/snippets/include/snippets/lowered/pass/insert_perf_count.hpp
        new file:   src/common/snippets/include/snippets/op/perf_count.hpp
        modified:   src/common/snippets/include/snippets/snippets_isa.hpp
        modified:   src/common/snippets/include/snippets/snippets_isa_tbl.hpp
        modified:   src/common/snippets/src/generator.cpp
        new file:   src/common/snippets/src/lowered/pass/insert_perf_count.cpp
        new file:   src/common/snippets/src/op/perf_count.cpp
        modified:   src/common/snippets/src/op/subgraph.cpp
        modified:   src/common/snippets/src/shape_inference/shape_inference.cpp
        modified:   src/common/snippets/tests/src/lowering_utils.cpp
        modified:   src/common/transformations/include/transformations/symbolic_transformations/dereshape_matmul.hpp
        modified:   src/common/transformations/src/transformations/convert_precision.cpp
        modified:   src/common/transformations/src/transformations/symbolic_transformations/dereshape_matmul.cpp
        modified:   src/common/transformations/src/transformations/symbolic_transformations/symbolic_optimizations.cpp
        new file:   src/common/transformations/tests/symbolic_transformations/dereshape_fullyconnected.cpp
        modified:   src/common/transformations/tests/utils/compress_quantize_weights.cpp
        modified:   src/core/dev_api/openvino/core/shape_util.hpp
        modified:   src/core/dev_api/openvino/core/so_extension.hpp
        deleted:    src/core/dev_api/tensor_conversion_util.hpp
        modified:   src/core/include/openvino/op/constant.hpp
        modified:   src/core/include/openvino/op/util/gather_base.hpp
        modified:   src/core/reference/include/openvino/reference/gather.hpp
        modified:   src/core/reference/src/op/function.cpp
        modified:   src/core/src/bound_evaluate.cpp
        modified:   src/core/src/model.cpp
        modified:   src/core/src/node.cpp
        modified:   src/core/src/op/abs.cpp
        modified:   src/core/src/op/acos.cpp
        modified:   src/core/src/op/acosh.cpp
        modified:   src/core/src/op/add.cpp
        modified:   src/core/src/op/asin.cpp
        modified:   src/core/src/op/asinh.cpp
        modified:   src/core/src/op/atan.cpp
        modified:   src/core/src/op/atanh.cpp
        modified:   src/core/src/op/ceiling.cpp
        modified:   src/core/src/op/clamp.cpp
        modified:   src/core/src/op/constant.cpp
        modified:   src/core/src/op/cos.cpp
        modified:   src/core/src/op/cosh.cpp
        modified:   src/core/src/op/cum_sum.cpp
        modified:   src/core/src/op/divide.cpp
        modified:   src/core/src/op/equal.cpp
        modified:   src/core/src/op/erf.cpp
        modified:   src/core/src/op/exp.cpp
        modified:   src/core/src/op/eye.cpp
        modified:   src/core/src/op/fake_convert.cpp
        modified:   src/core/src/op/fake_quantize.cpp
        modified:   src/core/src/op/floor.cpp
        modified:   src/core/src/op/floor_mod.cpp
        modified:   src/core/src/op/gather.cpp
        modified:   src/core/src/op/gelu.cpp
        modified:   src/core/src/op/greater.cpp
        modified:   src/core/src/op/greater_eq.cpp
        modified:   src/core/src/op/grid_sample.cpp
        modified:   src/core/src/op/hsigmoid.cpp
        modified:   src/core/src/op/hswish.cpp
        modified:   src/core/src/op/less.cpp
        modified:   src/core/src/op/less_eq.cpp
        modified:   src/core/src/op/log.cpp
        modified:   src/core/src/op/logical_not.cpp
        modified:   src/core/src/op/matmul.cpp
        modified:   src/core/src/op/max_pool.cpp
        modified:   src/core/src/op/maximum.cpp
        modified:   src/core/src/op/minimum.cpp
        modified:   src/core/src/op/mish.cpp
        modified:   src/core/src/op/mod.cpp
        modified:   src/core/src/op/multiply.cpp
        modified:   src/core/src/op/negative.cpp
        modified:   src/core/src/op/non_zero.cpp
        modified:   src/core/src/op/not_equal.cpp
        modified:   src/core/src/op/one_hot.cpp
        modified:   src/core/src/op/power.cpp
        modified:   src/core/src/op/prelu.cpp
        modified:   src/core/src/op/range.cpp
        modified:   src/core/src/op/reduce_l1.cpp
        modified:   src/core/src/op/reduce_l2.cpp
        modified:   src/core/src/op/reduce_logical_and.cpp
        modified:   src/core/src/op/reduce_logical_or.cpp
        modified:   src/core/src/op/reduce_max.cpp
        modified:   src/core/src/op/reduce_mean.cpp
        modified:   src/core/src/op/reduce_min.cpp
        modified:   src/core/src/op/reduce_prod.cpp
        modified:   src/core/src/op/reduce_sum.cpp
        modified:   src/core/src/op/relu.cpp
        modified:   src/core/src/op/round.cpp
        modified:   src/core/src/op/scatter_elements_update.cpp
        modified:   src/core/src/op/scatter_nd_update.cpp
        modified:   src/core/src/op/select.cpp
        modified:   src/core/src/op/sigmoid.cpp
        modified:   src/core/src/op/sign.cpp
        modified:   src/core/src/op/sin.cpp
        modified:   src/core/src/op/sinh.cpp
        modified:   src/core/src/op/softmax.cpp
        modified:   src/core/src/op/softplus.cpp
        modified:   src/core/src/op/softsign.cpp
        modified:   src/core/src/op/sqrt.cpp
        modified:   src/core/src/op/subtract.cpp
        modified:   src/core/src/op/swish.cpp
        modified:   src/core/src/op/tan.cpp
        modified:   src/core/src/op/tanh.cpp
        modified:   src/core/src/op/topk.cpp
        modified:   src/core/src/op/util/gather_base.cpp
        modified:   src/core/src/op/util/variable_value.cpp
        modified:   src/core/src/op/xor.cpp
        modified:   src/core/src/runtime/itensor.cpp
        modified:   src/core/src/shape_util.cpp
        deleted:    src/core/src/tensor_conversion_util.cpp
        modified:   src/core/tests/constant.cpp
        modified:   src/core/tests/tensor.cpp
        modified:   src/frontends/onnx/tests/__init__.py
        modified:   src/frontends/onnx/tests/runtime/ie/unit_test.manifest
        modified:   src/frontends/onnx/tests/tests_python/test_backend.py
        modified:   src/frontends/paddle/include/openvino/frontend/paddle/decoder.hpp
        modified:   src/frontends/paddle/include/openvino/frontend/paddle/node_context.hpp
        modified:   src/frontends/paddle/src/decoder_proto.cpp
        modified:   src/frontends/paddle/src/decoder_proto.hpp
        modified:   src/frontends/paddle/src/input_model.cpp
        modified:   src/frontends/paddle/src/input_model.hpp
        modified:   src/frontends/paddle/src/op/argmax.cpp
        modified:   src/frontends/paddle/src/op/elementwise_ops.cpp
        modified:   src/frontends/paddle/src/op/fill_constant_batch_size_like.cpp
        modified:   src/frontends/paddle/src/op/grid_sampler.cpp
        modified:   src/frontends/paddle/src/op/matmul_v2.cpp
        modified:   src/frontends/paddle/src/op/p_norm.cpp
        modified:   src/frontends/paddle/src/op/reduce_ops.hpp
        modified:   src/frontends/paddle/src/op/reverse.cpp
        modified:   src/frontends/paddle/src/op_table.cpp
        modified:   src/frontends/paddle/src/place.hpp
        modified:   src/frontends/paddle/src/proto/framework.proto
        modified:   src/frontends/paddle/tests/CMakeLists.txt
        modified:   src/frontends/paddle/tests/op_fuzzy.cpp
        modified:   src/frontends/paddle/tests/places.cpp
        modified:   src/frontends/paddle/tests/read_paddle_model_test.cpp
        modified:   src/frontends/paddle/tests/skip_tests_config.cpp
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_2in_2out.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_2in_2out_dynbatch.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_clip.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_conv2d_relu.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_conv2d_s.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_dynamic_pool2d.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_elementwise_ops.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_embedding.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_exp.py
        deleted:    src/frontends/paddle/tests/test_models/gen_scripts/generate_fill_constant.py
        deleted:    src/frontends/paddle/tests/test_models/gen_scripts/generate_fill_constant_batch_size_like.py
        new file:   src/frontends/paddle/tests/test_models/gen_scripts/generate_full.py
        renamed:    src/frontends/paddle/tests/test_models/gen_scripts/generate_fill_any_like.py -> src/frontends/paddle/tests/test_models/gen_scripts/generate_full_like.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_gelu.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_greater_equal.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_greater_than.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_grid_sampler.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_hard_sigmoid.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_hard_swish.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_index_select.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_layer_norm.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_leaky_relu.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_less_than.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_log.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_loop.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_mul.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_multi_tensor_split.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_place_test_model.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_pool2d.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_pow.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_prior_box.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_range.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_reduce_all.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_relu6.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_reshape.py
        deleted:    src/frontends/paddle/tests/test_models/gen_scripts/generate_reverse.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_scale.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_set_value.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_sigmoid.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_split.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_squeeze.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_strided_slice.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_swish.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_tile.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_unsqueeze.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_where.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/generate_where_index.py
        modified:   src/frontends/paddle/tests/test_models/gen_scripts/nms.py
        modified:   src/frontends/tensorflow/src/graph_iterator_meta.cpp
        modified:   src/frontends/tensorflow/src/graph_iterator_saved_model.cpp
        modified:   src/frontends/tensorflow/src/tf_utils.cpp
        modified:   src/frontends/tensorflow/tests/compilation.cpp
        modified:   src/inference/dev_api/openvino/runtime/threading/istreams_executor.hpp
        modified:   src/inference/dev_api/openvino/runtime/threading/thread_local.hpp
        modified:   src/inference/src/core.cpp
        modified:   src/inference/src/dev/threading/cpu_streams_executor.cpp
        modified:   src/inference/src/dev/threading/istreams_executor.cpp
        modified:   src/plugins/auto/src/plugin.cpp
        modified:   src/plugins/auto/src/plugin.hpp
        new file:   src/plugins/auto/tests/unit/stateful_model_test.cpp
        modified:   src/plugins/intel_cpu/src/compiled_model.cpp
        modified:   src/plugins/intel_cpu/src/config.cpp
        modified:   src/plugins/intel_cpu/src/cpu_streams_calculation.cpp
        modified:   src/plugins/intel_cpu/src/emitters/x64/cpu_generator.cpp
        modified:   src/plugins/intel_cpu/src/emitters/x64/jit_emitter.cpp
        modified:   src/plugins/intel_cpu/src/emitters/x64/jit_emitter.hpp
        new file:   src/plugins/intel_cpu/src/emitters/x64/jit_perf_count_chrono_emitters.cpp
        new file:   src/plugins/intel_cpu/src/emitters/x64/jit_perf_count_chrono_emitters.hpp
        new file:   src/plugins/intel_cpu/src/emitters/x64/jit_perf_count_rdtsc_emitters.cpp
        new file:   src/plugins/intel_cpu/src/emitters/x64/jit_perf_count_rdtsc_emitters.hpp
        modified:   src/plugins/intel_cpu/src/emitters/x64/jit_snippets_emitters.cpp
        modified:   src/plugins/intel_cpu/src/extension.cpp
        modified:   src/plugins/intel_cpu/src/nodes/reference.cpp
        modified:   src/plugins/intel_cpu/src/nodes/reshape.cpp
        modified:   src/plugins/intel_cpu/src/nodes/rope.cpp
        modified:   src/plugins/intel_cpu/src/nodes/rope.h
        modified:   src/plugins/intel_cpu/src/plugin.cpp
        modified:   src/plugins/intel_cpu/src/transformations/cpu_opset/common/op/rope.cpp
        modified:   src/plugins/intel_cpu/src/transformations/cpu_opset/common/op/rope.hpp
        modified:   src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/rope_fusion.cpp
        modified:   src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/rope_fusion.hpp
        new file:   src/plugins/intel_cpu/src/transformations/snippets/x64/op/perf_count_rdtsc.cpp
        new file:   src/plugins/intel_cpu/src/transformations/snippets/x64/op/perf_count_rdtsc.hpp
        modified:   src/plugins/intel_cpu/src/transformations/snippets/x64/shape_inference.cpp
        modified:   src/plugins/intel_cpu/src/utils/gen_pattern.hpp
        modified:   src/plugins/intel_cpu/src/utils/print_model.hpp
        modified:   src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/ctc_greedy_decoder.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/ctc_greedy_decoder_seq_len.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/ctc_loss.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/cum_sum.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/custom_op_internal_dyn.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/deformable_convolution.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/depth_to_space.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/detection_output.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/embedding_bag_offsets_sum.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/embedding_bag_packed_sum.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/embedding_segments_sum.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/extract_image_patches.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/eye.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/fake_quantize.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/select.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/shape_ops.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/shapeof.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/shuffle_channels.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/slice.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/space_to_batch.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/space_to_depth.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/split.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/strided_slice.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/tensor_iterator.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/tile.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/topk.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/unique.cpp
        modified:   src/plugins/intel_cpu/tests/functional/single_layer_tests/variadic_split.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/add_convert_to_reorder.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/align_matmul_input_ranks.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/any_layout.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/arm/convert_group_conv.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/arm/convert_group_conv1d.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/arm/convert_reduce_multi_axis.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/eltwise_caching.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/eltwise_chain.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/gather_add_avgpool.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/inplace_edge.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/input_noreorder_eltwise_bf16.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/input_output_tensor_reuse.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/input_tensor_roi.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/interaction.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/matmul_decompress_convert.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/matmul_quantized_subgraph.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/matmul_strided_inputs_outputs.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/matmul_weights_decompression.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/memory_sharing_test.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/mha.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/ngram.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/param_result_custom_blob.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/remove_convert.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/rotary_pos_emb.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/seq_native_order.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/split_concat_add.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/split_matmul_concat.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/static_zero_dims.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/strided_slice_zero_dims.cpp
        modified:   src/plugins/intel_cpu/tests/functional/subgraph_tests/src/subgraph_serialize.cpp
        modified:   src/plugins/intel_cpu/tests/functional/test_utils/arm/filter_cpu_info.cpp
        modified:   src/plugins/intel_cpu/tests/functional/test_utils/cpu_test_utils.cpp
        modified:   src/plugins/intel_cpu/tests/functional/test_utils/cpu_test_utils.hpp
        modified:   src/plugins/intel_cpu/tests/functional/test_utils/fusing_test_utils.cpp
        modified:   src/plugins/intel_cpu/tests/functional/test_utils/fusing_test_utils.hpp
        modified:   src/plugins/intel_cpu/tests/functional/test_utils/x64/filter_cpu_info.cpp
        modified:   src/plugins/intel_cpu/tests/unit/streams_info/streams_info_table_test.cpp
        modified:   src/plugins/intel_cpu/tests/unit/transformations/convert_to_rope.cpp
        modified:   src/plugins/intel_gpu/src/graph/include/loop_inst.h
        modified:   src/plugins/intel_gpu/src/graph/loop.cpp
        modified:   src/plugins/intel_gpu/src/kernel_selector/common_types.h
        modified:   src/plugins/intel_gpu/tests/unit/test_cases/scatter_update_gpu_test.cpp
        modified:   src/plugins/template/backend/int_executable.cpp
        modified:   src/plugins/template/backend/ops/if.cpp
        modified:   src/plugins/template/backend/ops/tensor_iterator.cpp
        modified:   src/tests/functional/plugin/conformance/test_runner/op_conformance_runner/src/read_ir/read_ir.cpp
        modified:   src/tests/functional/plugin/shared/src/behavior/ov_infer_request/io_tensor.cpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/activation.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/augru_cell.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/binary_convolution.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/comparison.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/convolution.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/convolution_backprop_data.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/dft.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/eltwise.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/embedding_bag_offsets_sum.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/embedding_bag_packed_sum.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/embedding_segments_sum.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/fully_connected.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/gather_nd.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/group_convolution.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/group_convolution_backprop_data.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/gru_cell.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/logical.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/lstm_cell.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/rdft.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/reduce.hpp
        modified:   src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/rnn_cell.hpp
        modified:   src/tests/test_utils/functional_test_utils/include/functional_test_utils/summary/op_summary.hpp
        modified:   src/tests/test_utils/functional_test_utils/layer_tests_summary/data/default_re_to_find_models.lst
        modified:   src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py
        modified:   src/tests/test_utils/functional_test_utils/layer_tests_summary/utils/stat_update_utils.py
        modified:   src/tests/test_utils/functional_test_utils/src/summary/op_summary.cpp
        modified:   tests/conditional_compilation/tools/infer_tool.py
        modified:   tests/layer_tests/mo_python_api_tests/test_mo_convert_pytorch.py
        modified:   tests/layer_tests/onnx_tests/test_reduce_lp.py
        modified:   tests/layer_tests/onnx_tests/test_roi_align.py
        modified:   tests/layer_tests/pytorch_tests/test_all.py
        modified:   tests/layer_tests/pytorch_tests/test_argmax_argmin.py
        modified:   tests/layer_tests/pytorch_tests/test_div.py
        modified:   tests/layer_tests/pytorch_tests/test_native_multi_head_attention.py
        modified:   tests/layer_tests/pytorch_tests/test_norm.py
        new file:   tests/layer_tests/tensorflow_tests/test_tf_AddTypes.py
        modified:   tests/layer_tests/tensorflow_tests/test_tf_ComplexFFT.py
        modified:   tests/layer_tests/tensorflow_tests/test_tf_MaxPoolWithArgmax.py
        modified:   tests/layer_tests/tensorflow_tests/test_tf_NormalizeL2.py
        modified:   tests/layer_tests/tensorflow_tests/test_tf_TopKV2.py
        modified:   tests/layer_tests/tensorflow_tests/test_tf_TruncateDiv.py
        modified:   tests/samples_tests/smoke_tests/test_speech_sample.py
        modified:   tests/stress_tests/common/ie_pipelines/pipelines.cpp
        modified:   tests/stress_tests/memcheck_tests/tests.cpp
        modified:   tests/time_tests/include/timetests_helper/utils.h
        modified:   tools/mo/openvino/tools/mo/utils/get_ov_update_message.py

PS C:\Users\Stark\Pictures\PythonOpenSourceContribution\openvino>_

Bhaskar365 commented:
> @Bhaskar365 there are still conflicts present - there are no new commits on you remote branch after the `git reset`. @akuporos's comment also remains unaddressed: [#21436 (comment)](https://github.com/openvinotoolkit/openvino/pull/21436#discussion_r1414346184).
> 
> `git reset` may have reverted some progress, if you've made any commits after the broken master merge.

Hello, [#21436 (comment)] was corrected by me yesterday but it seems every time the git reset is done , it is overriding my correction as well. Please instruct how to get this fixed as well with the current problem

p-wysocki commented:
`git status` returns a lot of files, it appears git got a bit messed up. Since working on a shared screen is not possible, I believe that the best option would be to delete the local repository on your machine altogether and clone it again. After that you can run:

`git fetch --all` to get your `splitFilesBranch`,

`git checkout `splitFilesBranch` to activate it. At this point you'll have clean git and can start working without worrying that past mistakes will influence current progress. After that's done you need to apply changes suggested by @akuporos again, sadly the previous commit has been lost.

Bhaskar365 commented:
Hi team,
          Please suggest me the list of procedure I would have to follow now.

Bhaskar365 commented:
Ok @p-wysocki , let me try that and I would let you know after I am done

Regards Bhaskar
